static/src/javascripts/projects/common/modules/discussion/comment-count.js

```diff
@@ -68,10 +72,15 @@ define([
                 var format,
                     $node = bonzo(node),
                     url = $node.attr('data-discussion-url') || getContentUrl(node),
+                    shortUrl = $node.attr('data-loyalty-short-url').replace('/p/','') || '',
                     $container,
                     meta,
                     html;
 
+                if (shortUrl && ab.isInVariant('ParticipationHideHalfOfComments', 'hide-comments') &&  CommentBlocker.hideComments(shortUrl) ) {
```

Comment:
@gtrufitt  Yeah. That's not possible in this case. This is the code that renders comment counts on container and front items and it runs way after the ab-test has run - injecting each of the comment counts into the dom. I guess we could use mediator and remove the  counts from the selected shortUrls, once commentcouints.js has run. This would mean that the user saw rendered comment counts and then saw some of them disappear :( Surely it's a better solution if the comment counts which are to be hidden don't appear and then disappear ..  You can't duplicate the shortUrl test in the canrun function because in this case, you're checking the shortUrl of each item in a container, or front. Come chat to me if this doesn't make sense